lib/core/src/Cardano/Wallet/DB.hs

```diff
@@ -381,19 +384,20 @@ cleanDB DBLayer{..} = atomically $
 -- Therefore, we need to keep the very first checkpoint in the database, no
 -- matter what.
 sparseCheckpoints
-    :: Quantity "block" Word32
+    :: SparseCheckpointsConfig
+        -- ^ Parameters for the function.
+    -> Quantity "block" Word32
```

Comment:
Since we know that `k` is static, wouldn't it be simpler if we moved it to inside `SparseCheckpointsConfig`?

Reply:
Makes sense to me :+1: